.pre-commit-config.yaml

```diff
@@ -0,0 +1,6 @@
+repos:
+- repo: https://github.com/astral-sh/ruff-pre-commit
+  rev: v0.3.7
+  hooks:
+  - id: ruff
+    args: [--fix, --exit-non-zero-on-fix]
```

Comment:
If you want ruff to format as well (to replace black), you need to add a format line to this.

Reply:
I kinda just wanted the check. Probably don't need it to format automatically